examples/hello-world/step-by-step/higgs/sklearn-linear/requirements.txt

```diff
@@ -1,4 +1,3 @@
-nvflare>=2.3.0
```

Comment:
I removed nvflare here because in the step before this pip, users have been asked to install nvflare properly